great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -507,9 +507,10 @@ def get_domain_records(
                     )
 
                 if ignore_row_if == "never":
+                    # deprecated-v0.13.29
                     warnings.warn(
                         f"""The correct "no-action" value of the "ignore_row_if" directive for the column pair case is \
-"neither" (the use of "{ignore_row_if}" will be deprecated).  Please update code accordingly.
+"neither" (the use of "{ignore_row_if}" is deprecated as of v0.13.29 and will be removed in v0.16).  Please update code accordingly.
```

Comment:
can we be more specific here in how the user gets on the non-deprecated path? "update accordingly" is a comment that will always inspire rage and frustration :)